backend/src/services/secret/secret-service.ts

```diff
@@ -1106,14 +1111,14 @@ export const secretServiceFactory = ({
       };
 
       // expand secrets
-      await batchSecretsExpand(decryptedSecrets);
+      await batchSecretsExpand(filteredSecrets);
```

Comment:
i don't think the names of the tags are set to their slugs here from the response

Reply:
That is secret expansion. We do the mapping in `decryptSecretRaw`

https://github.com/Infisical/infisical/pull/2242/files/bb52e2beb4567eb2eef9972ac90512fad3acb1a1#diff-9ad04b946494acd4d22dd0a4b21441235594fae872d62838717f57fdda3087d8R414

As this is used everywhere as end result in raw operation we will never miss the conversion